bigbio/hub/hub_repos/cei/cei.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from .bigbiohub import BigBioConfig, Tasks, text_features
+
+_DATASETNAME = "cei"
+_DISPLAYNAME = "CEI"
+
+_LANGUAGES = ["English"]
+_LOCAL = False
+_PUBMED = True
+
+_CITATION = """\
+@article{,
+  author    = {Larsson, Kristin and Baker, Simon and Silins, Ilona and Guo, Yufan and Stenius, Ulla and Korhonen, \
+    Anna and Berglund, Marika},
+  title     = {Text mining for improved exposure assessment},
+  journal   = {PloS one},
+  volume    = {12},
+  year      = {2017},
+  url       = {https://doi.org/10.1371/journal.pone.0173132},
+  doi       = {10.1371/journal.pone.0173132},
+  biburl    = {https://journals.plos.org/plosone/article/citation/bibtex?id=10.1371/journal.pone.0173132},
+  bibsource = {PloS one}
+}
+"""
+
+_DESCRIPTION = """\
+The Chemical Exposure Information (CEI) Corpus consists of 3661 PubMed publication abstracts manually annotated by \
+experts according to a taxonomy. The taxonomy consists of 32 classes in a hierarchy. Zero or more class labels are \
+assigned to each sentence in the corpus.
```

Comment:
the corpus does not really contain "sentences", but I guess the description was copied from the original source...